torch/csrc/jit/python/init.cpp

```diff
@@ -1504,7 +1504,19 @@ void initJITBindings(PyObject* module) {
           })
       .def_property_readonly(
           "is_mutable", [](FunctionSchema& self) { return self.is_mutable(); });
+
   py::class_<Argument>(m, "Argument")
+      .def("__repr__", [](Argument& self) {
+        std::ostringstream s;
+        if (self.alias_info() && !self.alias_info()->isWrite()) {
```

Comment:
Alban's suggestion:`
we should have a more pythonic print for Argument e.g.,`Argument(cpp_type='const Tensor&', name='self')`

maybe we should overload inputs print

Reply:
@ezyang what do you suggest?

Reply:
IDK, since you can't construct arguments by hand I don't think I care too much. I'd probably recommend lining up the C++ printing and the Python printing. There is an `operator<<` defined on Argument, you should use that.